README.md

```diff
@@ -8,10 +8,11 @@ This is a tool which enables running balenaOS docker images as a docker containe
 
 Before running this tool make sure:
 
-1. Docker daemon is running
-2. Current user has privileges to run docker commands.
-3. Depending on what balenaOS image you are trying to boot, you might need aufs or overlayfs on your host.
-4. The balenaOS image is compatible with the architecture where you are running the script. e.g. If you are running this script on your laptop (x86_64), you can run balenaOS images built for the NUC which are also x86_64. [***](#running-other-architectures)
+1. `balenaos-in-container` does not support [`cgroups v2`](https://www.kernel.org/doc/html/latest/admin-guide/cgroup-v2.html) yet. Check the version of `cgroups` of your machine. [Guides for Linux users](https://unix.stackexchange.com/questions/619681/how-can-i-find-out-what-version-of-cgroups-i-have), and [Docker Desktop users](...). Downgrade your `cgroups` to v1 if you want to use `balenaos-in-container`.
```

Comment:
How can we check if Docker Desktop uses cgroups v1 or v2?

Reply:
On macOS, assuming recent versions of Docker Desktop for macOS, file `~/Library/Group\ Containers/group.com.docker/settings.json` should have:
* Cgroup v1: "deprecatedCgroupv1": true,
* Cgroup v2: "deprecatedCgroupv1": false,

Docker Desktop must be stopped before making those changes.